spec/features/ctc/puerto_rico_spec.rb

```diff
@@ -26,7 +22,7 @@
 
   context "when the puerto rico intake has launched" do
     before do
-      allow(Flipper).to receive(:enabled?).and_return true
+      Flipper.enable :puerto_rico_home_location
```

Comment:
Do you have to explicitly un-enable the feature after the test completes? I'm wondering if this leaves the Flipper context where the feature is enabled as the rest of the tests run...